include/wlr/util/region.h

```diff
@@ -48,4 +50,10 @@ void wlr_region_expand(pixman_region32_t *dst, pixman_region32_t *src,
 void wlr_region_rotated_bounds(pixman_region32_t *dst, pixman_region32_t *src,
 	float rotation, int ox, int oy);
 
+void wlr_region_offset(pixman_region32_t *dst, pixman_region32_t *src,
```

Comment:
Does this translates regions?

Reply:
Doesn't `pixman_region32_translate` do the trick?

Reply:
It does... https://cgit.freedesktop.org/pixman/tree/pixman/pixman-region.c#n2220

Reply:
This has been resolved.